common/src/configuration/utils.rs

```diff
@@ -93,6 +93,10 @@ pub fn default_config(bootstrap: &ConfigBootstrap) -> Config {
     cfg.set_default("wallet.transaction_broadcast_send_timeout", 60)
         .unwrap();
     cfg.set_default("wallet.prevent_fee_gt_amount", true).unwrap();
+    cfg.set_default("wallet.transaction_sending_mechanism", "DirectAndStoreAndForward")
+        .unwrap();
+    cfg.set_default("wallet.command_send_wait_stage", "Broadcast").unwrap();
```

Comment:
Where are these two settings used?

Reply:
`wallet.command_send_wait_stage` is for the console wallet command/script mode. `wallet.transaction_sending_mechanism` is the new one used in this PR

Reply:
Not sure I understand the context of the question here. The simplistic answer is: `command_send_wait_stage` is used in the console wallet `pub async fn command_runner` and `transaction_sending_mechanism` is used in the console wallet's and base node wallet's bootstrap respectively if none are provided per config setting.